universal-login-sdk/lib/integration/ethereum/TokensDetailsStore.ts

```diff
@@ -6,10 +6,7 @@ export class TokensDetailsStore {
   constructor(private tokenDetailsService: TokenDetailsService, private tokensAddresses: string[]) {}
 
   async fetchTokensDetails() {
-    for (const address of this.tokensAddresses) {
-      const details = await this.tokenDetailsService.getTokenDetails(address);
-      this.tokensDetails.push(details);
-    }
+    this.tokensDetails = await Promise.all(this.tokensAddresses.map((address) => this.tokenDetailsService.getTokenDetails(address)));
```

Comment:
`Promise.all` -> almost always bad idea. If someone feels like it, we could create util, i.e.:
```ts
Promise.batched(5, Promise[])
` to improve performance, but it still would need to be low concurrency rate and configurable.